v3/library-script.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+
+# Necessary to stop pull requests from forks from running.
+if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then
+  npm run test:ci
```

Comment:
Correct, in a library's package.json.

Some of the libraries require a few "custom" steps before running either `skyux test` or `skyux build-public-library`. This will give the library control over what gets run during the "test" step.

See `@skyux/theme` as an example: https://github.com/blackbaud/skyux-theme/blob/master/package.json#L12

But, I can see where this might be a problem for sustainability. Let me think through this some more.

Reply:
I've replaced the package.json commands with SKY UX CLI commands.